src/util/git_repos.rs

```diff
@@ -641,10 +641,11 @@ mod tests {
             Uuid::from_str("6d80625b-5044-4aad-8d21-5d648371b52a").unwrap(),
             &TagQuery {
                 branch: Some(String::from("master")),
-                number: 1
+                number: 2
             }
         ).unwrap();
-        assert_eq!(commits, vec!["first"])
+        assert!(commits.contains(&String::from("first")));
```

Comment:
This seems extremely verbose.  Is this really how you have to make string constants in rust?

Reply:
Not exactly.  In this particular case, `commits` is a vector of Strings, so its `contains` method accepts only a borrowed String, as opposed to a &str, which is what `"first"` would be.